packages/discovery/src/cli/modelPermissionsCommand.ts

```diff
@@ -0,0 +1,63 @@
+import { Logger } from '@l2beat/backend-tools'
+import { TemplateService } from '../discovery/analysis/TemplateService'
+import { ConfigReader } from '../discovery/config/ConfigReader'
+import {
+  type DiscoveryPaths,
+  getDiscoveryPaths,
+} from '../discovery/config/getDiscoveryPaths'
+import { combinePermissionsIntoDiscovery } from '../discovery/modelling/combinePermissionsIntoDiscovery'
+import { modelPermissions } from '../discovery/modelling/modelPermissions'
+import { saveDiscoveredJson } from '../discovery/output/saveDiscoveryResult'
+import { sortEntry } from '../discovery/output/toDiscoveryOutput'
+import type { PermissionsOutput } from '../discovery/output/types'
+
+export async function modelPermissionsCommand(
```

Comment:
I'd rename this to just `modelPermissions`

Reply:
There's already `modelPermissions` so I want to avoid confusion. It also follows naming of the other files and functions in this folder: `discoverCommand()` and `singleDiscoveryCommand()`